Experimental/tests/Ranges/ImmutableRangeListTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+using NUnit.Framework;
+
+namespace CodeJam.Ranges
+{
+    [TestFixture]
+    public class ImmutableRangeListTests
+    {
+        [Test]
+        public void Contains()
+        {
+			// TODO:
+        }
+
+
+        [Test]
+        public void Invert()
+        {
+            var list = Range.Create(20, 30).ToRangeList().Add(Range.Create(11, 12, false, true));
+            CheckList(list, "[(11)..12, 20..30]");
```

Comment:
I thinks it is a bad idea to rely on string representations in tests. Later we could decide to change the string representation (for example, to make it more user-friendly) and we will have to change the tests although the main logic will not change at all.